source/docs/software/commandbased/binding-commands-to-triggers.rst

```diff
@@ -14,49 +14,42 @@ Trigger/Button Bindings
 
 There are a number of bindings available for the ``Trigger`` class. All of these bindings will automatically schedule a command when a certain trigger activation event occurs - however, each binding has different specific behavior. ``Button`` and its subclasses have bindings with identical behaviors, but slightly different names that better-match a button rather than an arbitrary triggering event.
 
-whenActive/whenPressed
-^^^^^^^^^^^^^^^^^^^^^^
+.. note:: Each type of binding has both ``True`` and ``False`` methods. For brevity, we'll be showing the ``True`` variants. The ``False`` methods are identical, with the reversal of ``true`` and ``false``.
```

Comment:
I think this note is unnecessary because there are statements in each section to discuss the difference.